perf/benchmark/run_benchmark_job.sh

```diff
@@ -260,36 +295,7 @@ for f in "${CONFIG_DIR}"/*; do
     
 done
 
-echo "collect flame graph ..."
-collect_flame_graph
-
-echo "perf benchmark test for istio is done."
-
-echo "start perf benchmark test for linkerd"
-# The following section is to run linkerd tests in the same cluster but within a different Namespace
-export NAMESPACE="twopods-linkerd"
-
-echo "Install Linkerd"
-pushd "${WD}/linkerd"
-./setup_linkerd.sh
-popd
-
-# setup linkerd test
-pushd "${WD}"
-export LINKERD_INJECT="enabled"
-./setup_test.sh
-popd
-
-export OUTPUT_DIR="linkerd_benchmark_data"
-LINKERD_LOCAL_OUTPUT_DIR="/tmp/${OUTPUT_DIR}"
-mkdir -p "${LINKERD_LOCAL_OUTPUT_DIR}"
-
-setup_fortio_and_prometheus
-
-CONFIG_DIR="${WD}/configs/linkerd"
-
-for f in "${CONFIG_DIR}"/*; do
-    get_benchmark_data "${f}"
-done
+#echo "collect flame graph ..."
+#collect_flame_graph
 
-echo "perf benchmark test for linkerd is done."
```

Comment:
I think comment out linkerd part is fine.

Reply:
we can add it back later, next step would be adding back CPU/mem after master and 1.5 greens